bentoml/_internal/yatai_rest_api_client/config.py

```diff
@@ -27,7 +27,9 @@ class YataiClientContext:
     name: str
     endpoint: str
     api_token: str
-    email: Optional[str] = attr.field(default=None)
+    email: Optional[str] = attr.field(
```

Comment:
why is this change? @aarnphm

Reply:
it was to fix the previous issue with yatai config, now I don't think we need to do this anymore.